napari/components/viewer_model.py

```diff
@@ -68,41 +81,50 @@ class ViewerModel(KeymapProvider, MousemapProvider):
         Contains axes, indices, dimensions and sliders.
     """
 
+    # After https://github.com/samuelcolvin/pydantic/pull/2196 is released
+    # make all of these fields with allow_mutation=False
+    axes: Axes = Axes()
+    camera: Camera = Camera()
+    cursor: Cursor = Cursor()
+    dims: Dims = Dims()
+    grid: GridCanvas = GridCanvas()
+    layers: LayerList = Field(
+        default_factory=LayerList
+    )  # Need to create custom JSON encoder for layer!
+    scale_bar: ScaleBar = ScaleBar()
+
+    active_layer: Optional[
+        Layer
+    ] = None  # Would be nice to remove this/ make it layer name instead of layer?
+    help: str = ''
+    status: str = 'Ready'
+    theme: str = DEFAULT_THEME
+    title: str = 'napari'
+
+    # 2-tuple indicating height and width
+    _canvas_size = (600, 800)
```

Comment:
```suggestion
    _canvas_size: Tuple[int, int] = (600, 800)
```

(and add typing import)